### PR TITLE
Add licensing information

### PR DESCRIPTION
**Pull request by skitt**
This documents the licenses we expect, and the exact header which our
CI checks for.

Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

**Review by Jaanki — APPROVED** [review]

File: src/content/development/licenses/_index.en.md
Comment (dfarrell07) [inline src/content/development/licenses/_index.en.md]:
```suggestion
weight: 35
```

To put it in ABC order.

**Review by dfarrell07 — CHANGES_REQUESTED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Review by nyechiel — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Comment by tpantelis** [comment]
hmm... why does it require a review from code owner @mkolesnik ... I'm a code owner (or should be) so that should be enough...

**Review by mkolesnik — APPROVED** [review]

**Comment by mkolesnik** [comment]
> hmm... why does it require a review from code owner @mkolesnik ... I'm a code owner (or should be) so that should be enough...

seems it likes me :)

**Pull request merged**
